stdlib/sys/file.gr

```diff
@@ -766,8 +752,101 @@ let wasmSafeCons = (a, b) => {
+                      wasmSafeCons(
+                        PathLinkSource,
+                        wasmSafeCons(
+                          PathLinkTarget,
+                          wasmSafeCons(
+                            PathOpen,
+                            wasmSafeCons(
+                              FdReaddir,
+                              wasmSafeCons(
+                                PathReadlink,
+                                wasmSafeCons(
+                                  PathRenameSource,
+                                  wasmSafeCons(
+                                    PathRenameTarget,
+                                    wasmSafeCons(
+                                      PathFilestats,
+                                      wasmSafeCons(
+                                        PathSetSize,
+                                        wasmSafeCons(
+                                          PathSetTimes,
+                                          wasmSafeCons(
+                                            FdFilestats,
+                                            wasmSafeCons(
+                                              FdSetSize,
+                                              wasmSafeCons(
+                                                FdSetTimes,
+                                                wasmSafeCons(
+                                                  PathSymlink,
+                                                  wasmSafeCons(
+                                                    PathRemoveDirectory,
+                                                    wasmSafeCons(
+                                                      PathUnlinkFile,
+                                                      wasmSafeCons(
+                                                        PollFdReadwrite,
+                                                        wasmSafeCons(
+                                                          SockShutdown,
+                                                          [],
+                                                        ),
+                                                      ),
+                                                    ),
+                                                  ),
+                                                ),
+                                              ),
+                                            ),
+                                          ),
+                                        ),
+                                      ),
+                                    ),
+                                  ),
+                                ),
+                              ),
+                            ),
+                          ),
+                        ),
+                      ),
+                    ),
+                  ),
+                ),
+              ),
+            ),
+          ),
+        ),
+      ),
+    ),
+  ),
+)
```

Comment:
lmfao

Reply:
:jinx:

Reply:
The formatter gives you a Christmas tree. :-D